### PR TITLE
create app

### PR DESCRIPTION
**Pull request by vladimirov-s**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    <div id="container">
+      <span>old version</span>
+      <div id="header">
+        <h1>To-do-lists</h1>
+        <input
+          type="text"
+          id="inpCreator"
+          class="posR"
+          placeholder="Наименование задачи"
+          title="Создание новой задачи"
```
Comment (baklanovairina) [inline index.html:29]: What is the difference between placeholder and title?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        />
+        <button id="but1" onclick="onClickButton()">
```
Comment (baklanovairina) [inline index.html:31]: but1 - incorrect name

Comment (baklanovairina) [reply index.html:31]: onClickButton - incorrect name

File: js/script.js [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+  }
+  cancelButton.onclick = () => {
+    container.removeChild(wrapForEdit);
+  }
+}
+const saveEdit = async (elem) => {
+  const { _id, text, isCheck } = elem;
+  if (elem.text) { elem.text = text; }
+  const response = await fetch(`${url}/updateTask`, {
+    method: "PATCH",
```
Comment (baklanovairina) [inline js/script.js:145]: Why patch?

File: js/script.js [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+      saveEdit(elem);
+    }
+  }
+  cancelButton.onclick = () => {
+    container.removeChild(wrapForEdit);
+  }
+}
+const saveEdit = async (elem) => {
+  const { _id, text, isCheck } = elem;
+  if (elem.text) { elem.text = text; }
```
Comment (baklanovairina) [inline js/script.js:143]: Firstly, if you write in one line, then {} is not needed. Secondly, it's better not to write in one line, it worsens the readability of the code.

File: js/script.js [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+    method: "PATCH",
+    headers: header,
+    body: JSON.stringify({ _id, text, isCheck })
+  });
+  const res = await response.json();
+  if (Array.isArray(res.data)) {
+    allTasks = res.data;
+    render();
+  } else {
+    console.warn(res.data);
```
Comment (baklanovairina) [inline js/script.js:154]: consoles should not be in the code

File: style/style.css [hunk 6]
```diff
@@ -0,0 +1,208 @@
...
+  text-align: center;
+  padding: 1rem 0 2rem 0;
+  max-width: 1200px;
+  min-width: 330px;
+  width: calc(100% - 1rem);
+  border-radius: 15px;
+  box-shadow: 0 0 15px 1px #000;
+  position: fixed;
+  left: calc(50% - 600px);
+  margin-top: -0.4rem;
```
Comment (baklanovairina) [inline style/style.css:24]: Why are there different units of measurement? Use just one. Either px or rem

File: style/style.css [hunk 7]
```diff
@@ -0,0 +1,208 @@
...
+}
+
+#container input {
+  font-size: 1.2rem;
```
Comment (baklanovairina) [inline style/style.css:28]: correct the units of measurement everywhere, choose one thing

File: style/style.css [hunk 8]
```diff
@@ -0,0 +1,208 @@
...
+  max-width: 100%;
+  top: -0.1rem;
+  background-color: #808080;
+}
+
+.posR {
+  position: relative;
+}
+
+.red-text {
```
Comment (baklanovairina) [inline style/style.css:119]: Choose one style for class names. Either already use camelCase everywhere or "-"

File: style/style.css [hunk 9]
```diff
@@ -0,0 +1,208 @@
...
+  color: red;
+}
+
+.text-task {
```
Comment (baklanovairina) [inline style/style.css:123]: Choose one style for class names. Either already use camelCase everywhere or "-"

File: js/script.js (hunk 3 above)
Comment (vladimirov-s) [reply js/script.js:145]: Because it works good

File: style/style.css (hunk 6 above)
Comment (vladimirov-s) [reply style/style.css:24]: because such was my measurement in the example.

**Pull request merged**